dll/win32/kernel32/winnls/string/japanese.c

```diff
@@ -0,0 +1,266 @@
+/*
+ * COPYRIGHT:       See COPYING in the top level directory
+ * PROJECT:         ReactOS system libraries
+ * FILE:            dll/win32/kernel32/winnls/string/japanese.c
+ * PURPOSE:         Japanese era support
+ * PROGRAMMER:      Katayama Hirofumi MZ
+ */
+
+#include <k32.h>
+
+#define NDEBUG
+#include <debug.h>
+#include "japanese.h"
+
+#define JAPANESE_ERA_MAX 16
+
+/* #define DONT_USE_REGISTRY */
```

Comment:
I'm assuming this will go away before merge?

Reply:
This code is useful for testing.